monitoring/README.md

```diff
@@ -194,7 +194,14 @@ You'll need to run a few more steps to completely setup the AAD Service Principa
 az keyvault set-policy --secret-permissions set --object-id $(az ad signed-in-user show --query id -o tsv) -n $ASB_KV_NAME -g $ASB_RG_CORE
 
 # Set grafana AAD client secrets
-az keyvault secret set -o table --vault-name $ASB_KV_NAME --name "grafana-aad-client-secret" --value "$CLIENT_SECRET"
+az keyvault secret set -o table --vault-name $ASB_KV_NAME --name "grafana-aad-client-secret" --value [insert CLIENT_SECRET]
```

Comment:
There are 2 `[insert CLIENT_SECRET]` sections here. Are they the same value? `CLIENT_SECRET` is set above in line 165, should that be used here?